### PR TITLE
tag: add {{No significant coverage (sports)}} and {{No significant coverage}}

### LINKED ISSUE #1817: tag: Add {{No significant coverage (sports)}}
I noticed that this template was missing from Twinkle, thus this issue.

### PR DESCRIPTION
**Pull request by earlt612**
Resolves #1817 and #2001

### CODE REVIEW

**Comment by Eejit43** [comment]
The template names have typos, they should be "signifi**c**ant" instead of "signifi**g**ant".

Additionally, more consistent descriptions would be:

`does not cite any sources containing significant coverage`
and
`sports biography that does not cite any sources containing significant coverage`

File: modules/friendlytag.js [hunk 1]
```diff
@@ -835,6 +835,8 @@ Twinkle.tag.article.tagList = {
 			{ tag: 'BLP sources', description: 'BLP that needs additional sources for verification' },
 			{ tag: 'BLP unsourced', description: 'BLP that has no sources at all (use BLP PROD instead for new articles)' },
 			{ tag: 'More citations needed', description: 'needs additional references or sources for verification' },
+			{ tag: 'No significant coverage', description: 'does not cite any sources containing significant coverage' },
+			{ tag: 'No significant coverage (sports)', 'sports biography that does not cite any sources containing significant coverage' },
```
Comment (NovemLinguae) [inline modules/friendlytag.js:839]: missing `description:`

**Review by NovemLinguae — CHANGES_REQUESTED** [review]

**Comment by earlt612** [comment]
@NovemLinguae: Requested changes have been completed

**Comment by NovemLinguae** [comment]
Thanks. I am finishing up a burst of work on https://github.com/wikimedia-gadgets/afc-helper. When I am done with that I will try to do a burst of work on Twinkle and catch up on all the PRs.

**Comment by earlt612** [comment]
No problem. Take your time

**Comment by NovemLinguae** [comment]
{{No significant coverage}} is fine and can go in.

{{No significant coverage (sports)}} - we should wait and see how the [TFD](Let's see how the TFD goes for this one: https://en.wikipedia.org/wiki/Wikipedia:Templates_for_discussion/Log/2024_September_4#Template:No_significant_coverage_(sports)) goes. even if the TFD passes, maybe we should decline to add that one. I might need to think about it.

**Review by NovemLinguae — APPROVED** [review]

**Comment by NovemLinguae** [comment]
The TFD is leaning keep. This PR has sat long enough. Let's approve it and get it in :)

**Pull request merged**